test/userTest.js

```diff
@@ -99,3 +99,32 @@ describe('Test for user Login', () => {
     expect(result.body).to.have.property('errors');
   });
 });
+
+describe('Test for update details', () => {
+  before(async () => {
+    await models.sequelize.sync({ force: true });
+  });
+  it('should create a new user', async () => {
+    const result = await supertest(app)
+      .post('/api/v1/users/signup')
+      .send({
+        username: 'kissess',
+        email: 'cosssys@coos.com',
+        password: 'aapppplee',
+      });
+    currentToken = result.body.user.token;
+    expect(result.status).to.be.equal(201);
+  });
+  it('should update user details', async () => {
+    const result = await supertest(app)
+      .patch('/api/v1/users')
+      .set('Authorization', currentToken)
+      .send({
+        username: 'mycheck',
```

Comment:
You have done a nice Job with your implementation. Since you updated the username in this test, I think you need to test for the username from the response object.

Reply:
Thanks, I am on it now